src/brevitas/graph/equalize.py

```diff
@@ -531,6 +652,7 @@ def __init__(
             module: nn.Module,
             weight_tensor_name: str,
             weight_axis: int,
+            act_axis: int,
```

Comment:
Maybe it should be? I'm not 100% sure though.
```suggestion
            act_axis: Optional[int],
```